.github/workflows/prepare_release.yml

```diff
@@ -0,0 +1,39 @@
+name: Prepare release
+
+on:
+  push:
+    tags: [ 'v*.*.*']
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.7'
+      - name: Set up dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install --upgrade setuptools wheel setuptools_scm build twine
+          python -m pip install -e .
+        shell: bash
+      - name: Build wheel
+        run: python -m build
+      - name: Verify build
+        run: twine check dist/*
+      - name: Create build archive
+        uses: a7ul/tar-action@v1.1.0
+        with:
+          command: c
+          files: dist
+          outPath: spdx_tools_dist.tar.gz
+      - name: Create GitHub release
```

Comment:
Can you add functionality to just create a prerelease instead of a full release for alpha, beta and rc

Reply:
As discussed, the releases are now created as drafts for simplicity, so that someone can look them over and mark them appropriately before publishing them.